INIT.md

```diff
@@ -1,10 +1,54 @@
-## webpack init
+# webpack-cli init
 
-Through [yeoman](http://yeoman.io/), the `webpack-cli init` feature allows people to create scaffolds and generate new projects quickly. An npm dependency that scaffolds a `webpack.config.js` through `webpack-cli` is what we refer to as an **addon**.
+`webpack-cli init` is used to initialize `webpack` projects quickly by scaffolding configuration and installing modules required for the project as per user preferences.
 
-We ask several questions in the default generator to get you started.
+## Initial Setup
+A. **Local setup**
 
----
+Follow given steps to locally setup `webpack-cli init` by installing dependencies:
+1. Create `package.json` through npm
+
+	```shell
+	$ npm init
+	```
+
+2. Install `webpack` and `webpack-cli` as devDependencies
+
+	```shell
+    $ npm install --save-dev webpack webpack-cli
+    ```
+
+3. Install @webpack-cli/init package to add init addon
+
+	```shell
+    $ npm install -D @webpack-cli/init
+    ```
+	
+B. **Global Setup**
+
+Follow following steps to setup `webpack-cli init` globally:
+1. Install `webpack` and `webpack-cli` globally
+	```shell
+	$ sudo npm install -g webpack webpack-cli
```

Comment:
Why doing `sudo`? Global modules are installed at user level, not root.

If you get errors in permission if you remove sudo then you probably have installed node/npm incorrectly in your local machine. Find the fix on Google. For now, remove `sudo` from here. :)

Reply:
Yes, I was getting errors,
Will remove `sudo` from both steps in next commit.

Reply:
@dhruvdutt , also I wasn't able to install `webpack-cli` globally using npm but `webpack` was installed.
Added this because it is only method (i.e. using -g flag) makes sense to me.

Reply:
What error are you getting?

Reply:
On running this:
`$ sudo npm install -g webpack-cli`
```console
npm WARN checkPermissions Missing write access to /usr/lib/node_modules/webpack-cli
npm WARN webpack-cli@3.0.8 requires a peer of webpack@^4.x.x but none is installed. You must install peer dependencies yourself.
```
I have webpack@4.16.1
But was able to setup webpack-command

Reply:
This issue is not related to webpack-cli. It happens because you've installed node/npm with incorrect permission. This is a common issue which I've also experienced it. You can Google it and find a fix.

Reply:
Yeah I was able to fix it by setting it up using nvm .